index.js

```diff
@@ -1658,6 +1665,31 @@
     return Functor.methods.map(functor)(f);
   }
 
+  //# flip :: Functor f => f (a -> b) -> a -> f b
+  //.
+  //. Apply a function contained in a Functor to a value.
+  //.
+  //. This function is derived from [`map`](#map).
+  //.
+  //. ```javascript
+  //. > function div(x) { return function(y) { return x / y }; }
```

Comment:
Not sure what a good example would be, especially considering that the API is a little weird without currying. I imagine Sanctuary would be able to provide a much nicer example for `flip` on Function.

Reply:
We're allowed to use arrow functions in doctests. How about the following?

```javascript
> flip(x => y => x + y, '!')('foo')
'foo!'
```

I prefer this example to the `div` example because `div` should really take the divisor as its first argument.